### ISSUE TITLE
Scanner

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-client#179]
Scanner von j-lawyer aus scans heraus aufrufen.... Bei mir ist der Scanner mit nem langen Fax belegt, ich könnte aber von j-lawyer aus scannen

**Comment by j-dimension** [comment]
Ich fürchte ich verstehe das Szenario noch nicht. Soll aus dem j-lawyer.org Client die Software eines Scanners gestartet werden?

**Comment by Mucki526** [comment]
Ja, das war die Idee... mit dem Ziel irgendwann alle Funktionen ausschließlich aus J-lawyer zu bedienen. Wieso z.B. auch der Umweg über einen Bereich Scan. Warum nicht im Dokumentordner einfach eine Taste Scan und die Scanfunktion packt den Scan direkt in die Akte in der ich bin. Das könnte ja recht einfach zu realisieren sein, weil ich ja im Moment nichts anderes mache als einen Scan einer Akte manuell zuzuordnen und den scan im scanordner wieder zu  löschen....

**Comment by j-dimension** [comment]
Das Schwierige ist die Fülle an unterschiedliche Scansoftware. Daher ist die Empfehlung, den Scanner in ein bestimmtes Verzeichnis scannen zu lassen. 

Mit der 1.9.1 würde nun auch (optional) das Löschen aus dem Scanordner entfallen - das kann wahlweise automatisch geschehen. Auch ein Tag kann bei Zuordnung automatisch gesetzt werden (bspw. "unbearbeiter Posteingang" o.ä.)
